specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,783 @@
+          "Terminate a Container Apps Job": {
+            "$ref": "./examples/ContainerAppsJob_Terminate.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerAppsJobs/{containerAppsJobName}/listSecrets": {
+      "post": {
+        "tags": [
+          "ContainerAppsJobs"
+        ],
+        "summary": "List secrets for a container apps job",
+        "operationId": "ContainerApps_ListSecrets",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "containerAppsJobName",
+            "in": "path",
+            "description": "Name of the Container Apps Job.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecretsCollection"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Container Apps Secrets": {
+            "$ref": "./examples/ContainerAppsJob_ListSecrets.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Configuration": {
+      "description": "Non versioned Container Apps Job configuration properties that define the mutable settings of a Container apps job",
+      "type": "object",
+      "required": [
+        "replicaTimeout",
+        "crashLoopRetryLimit",
+        "triggerType"
+      ],
+      "properties": {
+        "secrets": {
+          "description": "Collection of secrets used by a Container app",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/Secret"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "triggerType": {
+          "description": "Type of the job - Schedule | Event | Manual",
+          "enum": [
+            "Scheduled",
+            "Event",
+            "Manual"
+          ],
+          "type": "string",
+          "default": "Single",
+          "x-ms-enum": {
+            "name": "TriggerType",
+            "modelAsString": true
+          }
+        },
+        "replicaTimeout": {
+          "description": "Number of seconds from the startTime for each replica after which the system terminate it.",
```

Comment:
"Maximum number of seconds a replica is allowed to run." @vinisoto should we make this minutes or add a time unit parameter as well?